test/mock/rtps/RTPSReader/fastdds/rtps/reader/RTPSReader.h

```diff
@@ -177,6 +177,12 @@ class RTPSReader : public Endpoint
         return true;
     }
 
+    virtual bool matched_writers_guids(
+            std::vector<GUID_t>&) const
+    {
+        return true;
```

Comment:
Why this `return true` and the one in `RTPSWriter` returns `false`?

Reply:
True, will change it to `false`